docs/concepts/service-catalog/index.md

```diff
@@ -144,11 +144,11 @@ The following sequence diagram illustrates the steps involved in provisioning a
 
 1. When the `ServiceInstance` resource is created, Service Catalog initiates a *Provision Instance* call to the external Service Broker.
 1. The Service Broker creates a new instance of the Managed Service and returns an HTTP response.
-1. A {% glossary_tooltip text="Cluster Operator" term_id="cluster-operator" %} can then check the status of the instance to see if it is ready.
+1. A Cluster Operator can then check the status of the instance to see if it is ready.
 
 ### Binding to a Managed Service
 
-After a new instance has been provisioned, a {% glossary_tooltip text="Cluster Operator" term_id="cluster-operator" %} must bind to the Managed Service to get the connection credentials and service account details necessary for the application to use the service. This is done by creating a `ServiceBinding` resource.
+After a new instance has been provisioned, a Cluster Operator must bind to the Managed Service to get the connection credentials and service account details necessary for the application to use the service. This is done by creating a `ServiceBinding` resource.
 
 The following is an example of a `ServiceBinding` resource:
```

Comment:
Use sentence case for headings: Pod configuration file, not Pod Configuration File

Reply:
done